beacon-chain/cache/tracked_validators.go

```diff
@@ -0,0 +1,43 @@
+package cache
+
+import (
+	"sync"
+
+	"github.com/prysmaticlabs/prysm/v4/consensus-types/primitives"
+)
+
+type TrackedValidator struct {
+	Active       bool
+	FeeRecipient primitives.ExecutionAddress
+	Index        primitives.ValidatorIndex
```

Comment:
Do we really need an index here?

Reply:
not really, but it makes the code easier to read/write at very little cost.